Spanner/tests/Unit/DatabaseTest.php

```diff
@@ -947,8 +952,7 @@ public function testTransactionNestedTransaction()
                 )
             )
         ))
-            ->shouldBeCalled()
-            ->willReturn(['id' => self::TRANSACTION]);
+            ->shouldNotBeCalled();
```

Comment:
You can remove the rollback should not be called part since its anyways useless. 
Although you can keep the beginTransaction should not be called part.

Reply:
It was there already! I think they must have added for the following purpose: In normal workflow, if no `commit`/`rollback` is present in the callable passed to the runTransaction, `rollback` is automatically called. In this case it must not be called is what this test says. Hope it make sense to you.